SPECS/python-optree/python-optree.spec

```diff
@@ -30,18 +32,28 @@ A PyTree is a recursive structure that can be an arbitrarily nested Python conta
 
 
 %build
+
+export FFLAGS=$(echo $FFLAGS | sed 's/\(-Wp,-D_GLIBCXX_ASSERTIONS\)//g')
+export CXXFLAGS=$(echo $CXXFLAGS | sed 's/\(-Wp,-D_GLIBCXX_ASSERTIONS\)//g')
+export CFLAGS=$(echo $CFLAGS | sed 's/\(-Wp,-D_GLIBCXX_ASSERTIONS\)//g')
+export RPM_OPT_FLAGS=$(echo $RPM_OPT_FLAGS | sed 's/\(-Wp,-D_GLIBCXX_ASSERTIONS\)//g')
```

Comment:
Likewise for RPM_OPT_FLAGS and FCFLAGS.  I think it is safe to limit the change to just the CXXFLAGS or CFLAGS.  I believe it's using gcc not g++ to compile, so probably the CFLAGS are the only ones that actually matter.

Reply:
+1 to doing only surgical changes to the flags we actually need to change. Otherwise, the specs (or any piece of code, really) gets noisy, it's hard to understand the true intention and which changes we actually need. In general, never include redundant/unnecessary lines. It looks like you're very close now.

Reply:
updated